client/dashboard/sites/settings-ai-assistant/ai-assistant-form.tsx

```diff
@@ -116,6 +125,7 @@ export function AIAssistantForm( { site }: { site: Site } ) {
 	const siteEditorUrl = site?.URL + '/wp-admin/site-editor.php?canvas=edit';
 	const siteSpecUrl = site?.URL + '/wp-admin/site-editor.php?canvas=edit&ai-step=spec';
 	const mediaLibraryUrl = site?.URL + '/wp-admin/upload.php';
+	const postEditorUrl = site?.URL + '/wp-admin/edit.php';
```

Comment:
```suggestion
	const postEditorUrl = site?.URL + 'wp-admin/edit.php?post_type=post';
```

This would open the the `Posts` page specifically. Both `Posts` and `Pages` screens have the same route.

Reply:
Sure, by default it loads the post page without query param but fine to add that param too